#! /bin/sh /usr/share/dpatch/dpatch-run
## 55_ppd_okidata_name.dpatch by  <doko@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Forwarded upstream: http://www.cups.org/str.php?L2372

@DPATCH@
--- ./ppd/okidata9.ppd~	2005-12-15 17:19:46.000000000 +0100
+++ ./ppd/okidata9.ppd	2006-04-12 20:13:44.240749856 +0200
@@ -28,7 +28,7 @@
 *LanguageVersion: English 
 *LanguageEncoding: ISOLatin1
 *PCFileName:	"OKIDATA9.PPD"
-*Manufacturer:	"Oki"
+*Manufacturer:	"Okidata"
 *Product:	"(GNU Ghostscript)"
 *Product:	"(ESP Ghostscript)"
 *cupsVersion:	1.2
--- ./ppd/okidat24.ppd~	2005-12-15 17:19:46.000000000 +0100
+++ ./ppd/okidat24.ppd	2006-04-12 20:13:32.764494512 +0200
@@ -28,7 +28,7 @@
 *LanguageVersion: English 
 *LanguageEncoding: ISOLatin1
 *PCFileName:	"OKIDAT24.PPD"
-*Manufacturer:	"Oki"
+*Manufacturer:	"Okidata"
 *Product:	"(GNU Ghostscript)"
 *Product:	"(ESP Ghostscript)"
 *cupsVersion:	1.2
